core/trino-main/src/main/java/io/trino/sql/analyzer/StatementAnalyzer.java

```diff
@@ -415,7 +415,7 @@ protected Scope visitInsert(Insert insert, Optional<Scope> scope)
 
             for (ColumnMetadata column : columns) {
                 if (!accessControl.getColumnMasks(session.toSecurityContext(), targetTable, column.getName(), column.getType()).isEmpty()) {
-                    throw semanticException(NOT_SUPPORTED, insert, "Insert into table with column masks");
+                    throw semanticException(NOT_SUPPORTED, insert, "Insert into table with column masks is not supported");
```

Comment:
or "not allowed"

Reply:
This is at least consistent with the analogous message for `UPDATE`, see e.g. line 1866 and 1873

Reply:
```
kokosing@m16:~/presto$ grep -i 'not allowed' ./core/trino-main/src/main/java/io/trino/sql/analyzer/StatementAnalyzer.java
                        throw semanticException(INVALID_COLUMN_REFERENCE, reference, "LATERAL reference not allowed in %s JOIN", node.getType().name());
                            throw semanticException(COLUMN_NOT_FOUND, allColumns, "SELECT * not allowed from relation that has no columns");
                        throw semanticException(COLUMN_NOT_FOUND, allColumns, "SELECT * not allowed in queries without FROM clause");
                    throw semanticException(COLUMN_NOT_FOUND, allColumns, "SELECT * not allowed from relation that has no columns");
                    // Since RECURSIVE is specified, any reference to WITH query name is considered a recursive reference and is not allowed.
                            throw semanticException(INVALID_RECURSIVE_REFERENCE, recursiveReferences.get(0), "recursive reference not allowed in this context");
kokosing@m16:~/presto$ grep -i 'not supported' ./core/trino-main/src/main/java/io/trino/sql/analyzer/StatementAnalyzer.java
            throw semanticException(NOT_SUPPORTED, node, "USE statement is not supported");
                throw semanticException(NOT_SUPPORTED, insert, "Inserting into materialized views is not supported");
                throw semanticException(NOT_SUPPORTED, insert, "Inserting into views is not supported");
                throw semanticException(NOT_SUPPORTED, insert, "Insert into table with a row filter is not supported");
                    throw semanticException(NOT_SUPPORTED, insert, "Insert into table with column masks is not supported");
                throw semanticException(NOT_SUPPORTED, node, "Deleting from views is not supported");
                throw semanticException(NOT_SUPPORTED, node, "Analyzing views is not supported");
                throw semanticException(NOT_SUPPORTED, node, "Natural join not supported");
                throw semanticException(NOT_SUPPORTED, update, "Updating through views is not supported");
                throw semanticException(NOT_SUPPORTED, update, "Updating a table with a row filter is not supported");
                    throw semanticException(NOT_SUPPORTED, update, "Updating a table with column masks is not supported");
                // filter with window function is not supported yet
                    throw semanticException(NOT_SUPPORTED, windowFunction, "Window function with ORDER BY is not supported");
                    throw semanticException(NOT_SUPPORTED, allColumns, "Column aliases not supported");
                        throw semanticException(NOT_SUPPORTED, node.getSelect(), "SELECT * from outer scope table not supported with anonymous columns");
                    throw semanticException(EXPRESSION_NOT_IN_DISTINCT, expression, "Non deterministic ORDER BY expression is not supported with SELECT DISTINCT")
```

This is coherent with the current convention. I see your point so I created an issue for this, see: https://github.com/trinodb/trino/issues/7369